### DISCUSSION TITLE
Assets not included in search results

### DISCUSSION THREAD

**Discussion by pschocke** [statamic/cms#4656] [Q&A]
Hello everybody, I am seeking for your help:

I am trying to build a global search that includes some collections (pages and aktuelles) and also one assets container (downloads). They should be in one search result list.

However, I can only get the collections into the search results. No Asset is in the list.

My search.php:

```php
'indexes' => [
    'default' => [
        'driver' => 'local',
        'searchables' => ['collection:pages', 'collection:aktuelles', 'assets:downloads'],
        'fields' => ['title', 'content', 'navigation-title'],
      ],
],
```

I can verify that the asset is in the search/default.json search index:
```json
"asset::downloads::uebergabe_final.docx": {
    "title": "uebergabe_final.docx",
    "content": null,
    "navigation-title": null
}
```

However, if I use the search `search:results` tag and search for "uebergabe", nothing is shown.

my search (I am using blade and the blade directives package)

```blade
@foreach(tag('search:results') as $result)
    @dump($result)
@endforeach
```

The downloads container uses a simple local filesystem configuration:

```php
'disks' => [
    'downloads' => [
         'driver' => 'local',
         'root' => storage_path('app/public/downloads'),
         'url' => env('APP_URL').'/storage/downloads',
         'visibility' => 'public',
     ],
],
```

Is there anything I am missing? Thanks in advance for your time!

**Comment by pschocke** [comment]
After a lot of try and error I can see that my assets are included if I use the `search:results status:is=""` tag. However, than my other collections are not included in the results. Is there a way to add a status to my asset container so that it is understood in the same way Statamic interprets the status of collections?

**Reply by duncanmcclean** [reply]
This sounds like a bug to me - would you be able to [file a bug report](https://github.com/statamic/cms/issues/new?assignees=&labels=&template=bug_report.yml) for this?
